packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -225,7 +225,8 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
         }
         start = start ? start : (elementRange.start + (offset ? offset : 0)).toString();
         end = end ? end : (elementRange.end + (offset ? offset : 0)).toString();
-        const time = Math.round(elementRange.start + (offset ? offset : 0));
+        // use middle of state for fetching tooltip since hover time is not available
+        const time = Math.round((elementRange.start + elementRange.end) / 2 + (offset ? offset : 0));
```

Comment:
The formula is correct, but could (elementRange.start + elementRange.end) cause an overflow? Maybe the formular from below is safer (?):
Math.round((elementRange.start + ((elementRange.end - elementRange.start) / 2)) + (offset ? offset : 0))